specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -564,6 +572,223 @@
+        "operationId": "Weather_GetAirQualityDailyForecast",
+        "x-ms-examples": {
+          "Get Daily Air Quality Forecast": {
+            "$ref": "./examples/GetDailyAirQuality.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/JsonFormat"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/Language"
+          },
+          {
+            "name": "query",
+            "description": "Coordinates of the location for which air quality is requested. The applicable query is specified as a comma separated string composed by latitude followed by longitude e.g. \"47.641268,-122.125679\".",
+            "in": "query",
+            "required": true,
+            "x-ms-client-name": "coordinates",
+            "x-ms-parameter-location": "method",
+            "type": "array",
+            "collectionFormat": "csv",
+            "minItems": 2,
+            "maxItems": 2,
+            "items": {
+              "type": "number",
+              "format": "double"
+            }
+          },
+          {
+            "name": "duration",
+            "description": "Specifies for how many days from now we would like to know about the air quality. Available values are 1, 2, 3, 4, 5, 6, and 7. Default value is 1.",
+            "default": 1,
+            "in": "query",
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 7
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DailyAirQualityForecastResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/airQuality/forecast/hourly/{format}": {
+      "get": {
+        "description": "**Get Air Quality Hourly Forecast**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.n\nGet detailed information about the concentration of pollutants and overall status for forecasted upcoming hourly air quality. The service can provide forecasted hourly air quality information for the upcoming time spans of 1, 12, 24, 48, 72, and 96 hours. Information includes, pollution levels, air quality index values, the dominant pollutant, and a brief statement summarizing risk level and suggested precautions.",
```

Comment:
There is a spacing issue in the text.

Change:
.n\nGet

To:
. \n\nGet

Reply:
Updated